code/modules/mob/mob.dm

```diff
@@ -386,9 +386,6 @@
 	if(I)
 		I.attack_self(src)
 		update_inv_hands()
-	if(!I)//CIT CHANGE - allows "using" empty hands
-		use_that_empty_hand() //CIT CHANGE - ditto
-		update_inv_hands() // CIT CHANGE - ditto.
```

Comment:
What's the reason for removing this?

Reply:
I either had to delete this unused code or cut paste it elsewhere so I could also delete the mob.dm file, and since I doubt any feature will ever use `use_that_empty_hand()`, I opted in for the first option. Anyway sorry for not including it in the PR description, it's a minor bit I forgot about.